tests/test_sequence.py

```diff
@@ -139,9 +139,9 @@ def test_magnetic_field():
 
 
 @pytest.fixture
-def test_devices():
+def devices():
 
-    test_device1 = Device(
+    device1 = Device(
```

Comment:
This is still a `Device` and you're calling `change_rydberg_level()` on it. Use a `VirtualDevice` instead

Reply:
I am not calling `change_rydberg_level()` anymore

Reply:
You're right, my bad